js/client.js

```diff
@@ -0,0 +1,280 @@
+//TODO: Consider using JS modules for imports
+
+//TODO: Consider splitting game logic into separate code from base
+// client?
+
+// *******  Constants   ******
+const bgColor= "black";
+const strokeStyle = "white";
+
+//The matrices representing the game pieces
+const pieceZ = [ 1, 1, 0, 0, 1, 1, 0, 0, 0]; //0
+const pieceS = [ 0, 1, 1, 1, 1, 0, 0, 0, 0]; //1
+const pieceJ = [ 1, 0, 0, 1, 1, 1, 0, 0, 0]; //2
+const pieceR = [ 0, 0, 1, 1, 1, 1, 0, 0, 0]; //3
+const pieceT = [ 0, 1, 0, 1, 1, 1, 0, 0, 0]; //4
+const pieceI = [ 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0]; //5
+const pieceO = [ 0, 0, 0, 0, 0, 1, 1, 0, 0, 1, 1, 0, 0, 0, 0, 0]; //6
+const shapes = [pieceZ, pieceS, pieceJ, pieceR, pieceT, pieceI, pieceO];
+
+// ******   Variables   ********
+//TODO: Consider removing global variables or moving relevant parts to
+//other components
```

Comment:
I agree with your comment here -- globals aren't great, but in this instance they aren't hurting anyone :)